src/api/profile/mock/profile.ts

```diff
@@ -0,0 +1,16 @@
+import type { ProfileModel } from '../types';
+
+const profile: ProfileModel = {
+  id: '0',
+  photo: '../../../assets/images/profile-mock.jpg',
```

Comment:
```suggestion
import profileMockImage from 'assets/images/profile-mock.jpg';
import type { ProfileModel } from '../types';

const profile: ProfileModel = {
  id: 0,
  photo: profileMockImage,
```

Без предварительного импорта картинка не отобразится + id поменять на число